README.md

```diff
@@ -29,7 +29,13 @@ This library is GWT-compatible; if you use this with GWT, add the following to y
 ```xml
 <inherits name='make.some.noise' />
 ```
-
+and also add the following to your `html/build.gradle` `dependencies {}` section:
+```groovy
+dependencies {
+    ...
+    implementation "com.github.tommyettinger:make_some_noise:0.3:sources"
```

Comment:
```suggestion
    implementation "com.github.tommyettinger:make_some_noise:0.4:sources"
```

Reply:
Ah releasing a new version after my pr. "rude"! :D 
Thanks for the suggestion!